api_app/script_analyzers/observable_analyzers/stratosphere.py

```diff
@@ -0,0 +1,140 @@
+import os
+import logging
+import traceback
+import requests
+
+from api_app.exceptions import AnalyzerRunException
+from api_app.script_analyzers import classes
+from intel_owl import settings
+from datetime import datetime, date
+
+logger = logging.getLogger(__name__)
+
+db_name0 = "stratos_ip_blacklist_last24hrs.csv"
+db_name1 = "stratos_ip_blacklist_new_attacker.csv"
+db_name2 = "stratos_ip_blacklist_repeated_attacker.csv"
+
+db_loc0 = f"{settings.MEDIA_ROOT}/{db_name0}"
+db_loc1 = f"{settings.MEDIA_ROOT}/{db_name1}"
+db_loc2 = f"{settings.MEDIA_ROOT}/{db_name2}"
+
+
+class Stratos(classes.ObservableAnalyzer):
+    def run(self):
+        ip = self.observable_name
+        result = {
+            "last24hrs": False,
+            "last24hrs_rating": "_",
+            "new_attacker": False,
+            "new_attacker_rating": "_",
+            "repeated_attacker": False,
+            "repeated_attacker_rating": "_",
+        }
+
+        self.check_dataset_status()
+
+        with open(db_loc0, "r") as f:
```

Comment:
please try to avoid the duplication of code for this

Reply:
Changed the code to have minimum duplication.